projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -83,6 +85,11 @@ type PromptOptionsProps = {
 	 */
 	content: string;
 
+	/*
+	 * The delimited content.
+	 */
+	delimitedContent: string;
```

Comment:
Can we avoid adding new props to the helper functions?
Could you add the delimiter changes straight into the fn?

Reply:
```jsx
content = `${ delimiter }${ content?.replaceAll(
		delimiter,
		''
	) }${ delimiter }`;
```

Something like this